posts/2020-09-01-jakarta-microprofile-beta-200010.adoc

```diff
@@ -0,0 +1,355 @@
+---
+layout: post
+title: "New features for MicroProfile 4.0 and more Jakarta EE 9 features in Open Liberty 20.0.0.10 beta"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/jakub-pomykala
+author_github: https://github.com/jakub-pomykala
+seo-title: New features for MicroProfile 4.0 and more Jakarta EE 9 features in Open Liberty 20.0.0.10 beta - OpenLiberty.io
+seo-description: Open Liberty 20.0.0.10 beta comes with new features for MicroProfile 4.0 and more Jakarta EE 9 features. Give it a try by adding either Jakarta EE 9 Beta features package coordinates or the All Beta features package coordinates to your build tools.
+blog_description: Open Liberty 20.0.0.10 beta comes with new features for MicroProfile 4.0 and more Jakarta EE 9 features. Give it a try by adding either Jakarta EE 9 Beta features package coordinates or the All Beta features package coordinates to your build tools.
+---
+= New features for MicroProfile 4.0 and more Jakarta EE 9 features in Open Liberty 20.0.0.10 beta
+Jakub Pomykala <https://github.com/jakub-pomykala>
+:imagesdir: /
+:url-prefix:
+:url-about: /about/
+
+In this release of Open Liberty Beta you can try the new features for MicroProfile 4.0 which are Fault Tolerance 3.0, Matric 3.0, Health 3.0 and OpenTracing 2.0 plus more Jakarta EE 9 features.
+
+We have two beta packages for link:{url-about}[Open Liberty]:
```

Comment:
I recommend removing the link here completely, but at the very least change it to / instead of /about/, as the latter just redirects to the former.  Also, update the release template to do the same

Reply:
Done